### PR TITLE
Updated Android SDK version

### PR DESCRIPTION
**Pull request by orestesgaolin**
There was issue with previous version of Android SDK that it used wrong text colors for dark backgrounds. I got response from Intercom support team, that updating to 12.5.1 should solve that.

### CODE REVIEW

File: intercom_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.4.1
+
+* Bump Intercom Android SDK version to 12.5.1
```
Comment (Zazo032) [inline intercom_flutter/CHANGELOG.md]:
```suggestion
* Bump Intercom Android SDK version to 12.5.1 [(#261)](https://github.com/v3rm0n/intercom_flutter/pull/261)
* Android 13 support
```

**Review by Zazo032 — CHANGES_REQUESTED** [review]
🥇 Thanks for the contribution!
Since version 12.5.0, Android 13 is now officially supported by Intercom, I think we should also make some small changes to the README.md file:

- In the optional permissions for Android section, add:
```xml
<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```
- In the Push notifications setup section, add extra steps for Android setup:
  - Steps for Android notifications setup: https://developers.intercom.com/installing-intercom/docs/android-fcm-push-notifications
  - We could also add that for push notifications to work, user might have to grant the notification permission in iOS or Android 13+ with a plugin like [permission_handler](https://pub.dev/packages/permission_handler)

**Comment by orestesgaolin** [comment]
Thanks, will apply the changes :)

**Comment by orestesgaolin** [comment]
I updated the README.md and from what I can see right now the `firebase_messaging` should take care of the notification permission, so as this is a prerequisite it seems we don't need to ask users to install `permission_handler`, wdyt?

**Review by Zazo032 — APPROVED** [review]
You're right, I didn't notice there was a new update for `firebase_messaging` that automatically handled that for Android too. Changes LGTM, thanks for you help!

**Pull request merged**